### PR TITLE
[RELENG-5948] Remove openstack hardcoded settings and add portability

### PR DESCRIPTION
**Pull request by erwan-b**

### CODE REVIEW

**Review by tcarmet — APPROVED** [review]

File: config_example.yml [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+    token: ""
+    network_name: ""
+
+# SSH key added on each runner
 allowed_ssh_keys: ""
 
-# RedHat credentials
-redhat_username: ""
-redhat_password: ""
-
```
Comment (tcarmet) [inline config_example.yml:12]: Why are we removing this?

Comment (erwan-b) [reply config_example.yml:12]: This is env variables for now

File: srcs/runners_manager/main.py [hunk 2]
```diff
@@ -26,17 +26,19 @@ def maintain_number_of_runner(runner_m: Manager, github_manager: GithubManager):
         time.sleep(10)
 
 
+def get_cloud_manager(settings: dict, args: EnvSettings) -> CloudManager:
+    CloudManager = importlib.import_module(f'runners_manager.vm_creation.{settings["cloud_name"]}')
+
+    return CloudManager.CloudManager(openstack_settings=settings['cloud_config'],
```
Comment (tcarmet) [inline srcs/runners_manager/main.py:32]:
```suggestion
    return CloudManager.CloudManager(cloud_config=settings['cloud_config'],
```

File: srcs/runners_manager/main.py [hunk 3]
```diff
@@ -26,17 +26,19 @@ def maintain_number_of_runner(runner_m: Manager, github_manager: GithubManager):
...
     importlib.import_module(settings['python_config'])
-    openstack_manager = OpenstackManager(project_name=settings['cloud_nine_tenant'],
-                                         region=settings['cloud_nine_region'],
-                                         token=args.cloud_nine_token,
-                                         username=args.cloud_nine_user,
-                                         password=args.cloud_nine_password,
-                                         redhat_username=args.redhat_username,
-                                         redhat_password=args.redhat_password,
-                                         ssh_keys=settings['allowed_ssh_keys'])
+    openstack_manager = get_cloud_manager(settings, args)
```
Comment (tcarmet) [inline srcs/runners_manager/main.py:41]:
```suggestion
    cloud_manager = get_cloud_manager(settings, args)
```

**Review by tcarmet — APPROVED** [review]
LGTM just ensure we're not forgetting any openstack reference in the rest of the code to ensure all the naming and code structure are cloud agnostic

**Pull request merged**